modules/gdscript/gdscript_vm.cpp

```diff
@@ -2545,6 +2555,56 @@ Variant GDScriptFunction::call(GDScriptInstance *p_instance, const Variant **p_a
 			}
 			DISPATCH_OPCODE;
 
+			OPCODE(OPCODE_JUMP_IF_NULL) {
+				CHECK_SPACE(4);
+
+				GET_VARIANT_PTR(src, 0);
+				GET_VARIANT_PTR(dst, 1);
+
+				bool should_skip = false;
+				if (src->get_type() == Variant::NIL) {
+					should_skip = true;
+				} else if (src->get_type() == Variant::OBJECT) {
+					Object *obj = src->get_validated_object_with_check(should_skip);
+					should_skip |= obj == nullptr;
```

Comment:
Does this simplify to `should_skip = src->get_validated_object();`? `OPCODE_EXIT_IF_NULL` is the same.